### PR TITLE
Add import export support of python content

### LINKED ISSUE #609: Add import export support of python content
## Summary
Add import export support of python content

### PR DESCRIPTION
**Pull request by tjmullicani**
Review Checklist:
- [ ] An issue is properly linked. [feature and bugfix only]
- [ ] Tests are present or not feasible.
- [ ] Commits are split in a logical way (not historically).

fixes #609

### CODE REVIEW

**Comment by tjmullicani** [comment]
requires https://github.com/pulp/pulp-cli/issues/609

File: pulpcore/cli/python/context.py [hunk 1]
```diff
@@ -71,6 +71,7 @@ class PulpPythonRepositoryContext(PulpRepositoryContext):
     ENTITIES = _("python repositories")
     ID_PREFIX = "repositories_python_python"
     VERSION_CONTEXT = PulpPythonRepositoryVersionContext
+    CAPABILITIES = {"pulpexport": [PluginRequirement("python")]}
```
Comment (mdellweg) [inline pulpcore/cli/python/context.py:74]: This needs to include the version of pulp_python to include this feature.

Comment (tjmullicani) [reply pulpcore/cli/python/context.py:74]: Ok, we have to wait for https://github.com/pulp/pulp_python/pull/580 to be included in the next pulp_python release.

File: CHANGES/609.feature [hunk 2]
```diff
@@ -0,0 +1 @@
+Add import export support of python content
```
Comment (mdellweg) [inline CHANGES/609.feature:1]:
```suggestion
Added import export support of python content.
```

File: pulp-glue/pulp_glue/python/context.py [hunk 3]
```diff
@@ -88,7 +88,10 @@ class PulpPythonRepositoryContext(PulpRepositoryContext):
     ENTITIES = _("python repositories")
     ID_PREFIX = "repositories_python_python"
     VERSION_CONTEXT = PulpPythonRepositoryVersionContext
-    CAPABILITIES = {"sync": [PluginRequirement("python")]}
+    CAPABILITIES = {
+        "sync": [PluginRequirement("python")],
+        "pulpexport": [PluginRequirement("python")],
```
Comment (mdellweg) [inline pulp-glue/pulp_glue/python/context.py:93]: I'm almost certain, this needs a version dependency (unless it's >=3.1.0).

**Comment by mdellweg** [comment]
Marking as draft while we wait on the server plugin parts.

We would love to see you coming back on these.

**Comment by gerrod3** [comment]
Completed in https://github.com/pulp/pulp-cli/pull/887

**Pull request closed without merging**